# input.yaml
projector_type: "MGRS"
mgrs_grid: "54SUE"